peekingduck/pipeline/nodes/draw/utils/general.py

```diff
@@ -59,4 +59,4 @@ def project_points_onto_original_image(
     projected_points[:, 0] *= width
     projected_points[:, 1] *= height
 
-    return projected_points
+    return projected_points.astype(int)
```

Comment:
Just a note that `numpy.astype()` does not round up the numbers:
```
>>> import numpy as np
>>> x = np.array([1.1, 2.5, 3.7])
>>> x
array([1.1, 2.5, 3.7])
>>> x.astype(int)
array([1, 2, 3])
```

If intention is to round, can do this:
```
>>> np.round(x).astype(int)
array([1, 2, 4])
```

Reply:
Implemented rounding for that extra bit of accuracy.